roles/container_runtime/tasks/package_crio.yml

```diff
@@ -51,11 +51,56 @@
     - /etc/cni/net.d/200-loopback.conf
     - /etc/cni/net.d/100-crio-bridge.conf
 
-- name: Create the CRI-O configuration
-  template:
-    dest: /etc/crio/crio.conf
-    src: crio.conf.j2
-    backup: yes
+- name: Set crio.conf storage_driver
+  ini_file:
+    path: /etc/crio/crio.conf
+    section: crio
+    option: storage_driver
+    value: '"overlay"'
+
+# The ending ']' is absent in several values below because the ending ']' is on
```

Comment:
This looks error-prone to me.

I would prefer sticking with a template, at least then we know we're working with something sensible.  Otherwise, we need a module that supports this wacky syntax.